core/tests/webdriverio_utils/SkillEditorPage.js

```diff
@@ -234,6 +240,16 @@ var SkillEditorPage = function() {
     expect(await saveOrPublishSkillButton.isEnabled()).toEqual(false);
   };
 
+  this.updateQuestion = async function(commitMessage) {
```

Comment:
Should this be saveChangesToQuestion()? 

"Update" feels like includes the whole editing process as well.